Makefile

```diff
@@ -95,8 +95,9 @@ test-unit: setup-envtest ## Run the unit tests
 	eval $$($(SETUP_ENVTEST) use -p env $(ENVTEST_VERSION)) && go test -tags $(GO_BUILD_TAGS) -count=1 -short $(UNIT_TEST_DIRS)
 
 FOCUS := $(if $(TEST),-v -focus "$(TEST)")
+IMAGE_REGISTRY ?= "testdata"
```

Comment:
I expected to see some changes in `kind-load-bundles` to reflect the fact that the registry name is now coming from this variable.

Also, should we go ahead and make this `IMAGE_REPO ?= "testdata/bundles"`? It feels like the "bundles" repo name is not something we can always count on containing the images we want.